app/uk/gov/hmrc/cdsimportsddsfrontend/controllers/forms/DocumentationTypeFormMapping.scala

```diff
@@ -16,9 +16,9 @@
 
 package uk.gov.hmrc.cdsimportsddsfrontend.controllers.forms
 
-import play.api.data.Forms.{mapping, optional, text}
+import play.api.data.Forms.{mapping, optional, text, seq}
 import play.api.data.Mapping
-import uk.gov.hmrc.cdsimportsddsfrontend.domain.DocumentationType
+import uk.gov.hmrc.cdsimportsddsfrontend.domain._
 
 object DocumentationTypeFormMapping {
```

Comment:
This object name should not have the word `Type` in it

The CDS PDF name for section 2 is `References` so this object should be called `ReferencesFormMapping`

Reply:
I'll do this as part of a refactor ticket.